sc/social/like/subscribers.py

```diff
@@ -107,3 +113,49 @@ def assign_canonical_url(obj, event):
             'Canonical domain not set in Social Media configlet; '
             "Facebook's Open Graph canonical URL (og:orl) will not be available"
         )
+
+
+def social_content_check(obj, event):
+    """Check if content is valid if enabled in sc.social.content, in action when edit."""
+
+    request = obj.REQUEST
+    try:
```

Comment:
can we use the add-on browser layer instead?

Reply:
I have not tried I used base the 'amp'